src/Test.UnitTests.Sarif.Multitool/ExtensionsTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    public class ExtensionsTests
+    {
+        [Theory]
+        [InlineData(-1, "", "", true)]
```

Comment:
>InlineData [](start = 9, length = 10)

Need a test where `guid` is `null`. That is actually the more likely case -- the SARIF file doesn't even _mention_ `guid`. It is less likely that `guid` is present and its value is the empty string. Keep the "empty string" test, though, by all means.